features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumMonitor.kt

```diff
@@ -20,6 +20,11 @@ import com.datadog.tools.annotation.NoOpImplementation
 @Suppress("ComplexInterface", "TooManyFunctions")
 @NoOpImplementation
 interface RumMonitor {
+    /**
+     * The current active session id. May return null if a session has not been started yet.
+     * Will return a "null session id" (all zeros) if the current session is not tracked.
```

Comment:
To be precise, it won't be all zeros, but `00000000-0000-0000-0000-000000000000`, so user cannot write something like `if (currentSessionId.all {it == '0'})`, so maybe for the sake of comparison with `null session` mentioned in the documentation, we need to expose such value?